node/integrationtest/yarn.js

```diff
@@ -102375,7 +102375,7 @@ function bufferFrom (value, encodingOrOffset, length) {
 
   return isModern
     ? Buffer.from(value)
-    : new Buffer(value)
+    : Buffer.from(value)
```

Comment:
It looks like both sides of the ternary are the same now. Based on the `isModern` variable, my guess is that this change should be backed out.

Reply:
Thanks, reverted.